### ISSUE TITLE
Missing si_T torque conversion from lbm to si units

### ISSUE THREAD

**Issue by trparry** [ProjectPhysX/FluidX3D#38]
I think we are missing si_T, to convert torque from lbm units to si units. Its probably just adding the line 
`	float si_T(const float T) const { return T * kg * sq(m) / sq(s); } //torque si_T = T*[kg*m^2/s^2]`
to units.hpp.

**Comment by ProjectPhysX** [comment]
Yes, thank you! I just added the conversion functions to the repository.
